index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+    ul {
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+    }
+    p{
+      font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+      font-size: large;
+      margin-left: 15%;
+      margin-right: 15%;
+      padding-top: 15%;
+    }
+    #welcome-section{
+      height: 100vh;
+    }
+    #navbar {
+  overflow: hidden;
+  background-color: lightyellow;
+  position: fixed; /* Set the navbar to fixed position */
+  top: 0; /* Position the navbar at the top of the page */
+  width: 100%; /* Full width */
+    }
+@media only screen and (max-width: 4180px) {
+  body {
```

Comment:
good Job so far @lu-jim but there is some change that needs to be done 

you need to remove inline styles